hw/top_earlgrey/ip/rstmgr/data/autogen/rstmgr.hjson

```diff
@@ -226,14 +220,6 @@
         },
 
         { bits: "2",
-          name: "NDM_RESET",
```

Comment:
Wait, so we have no bit that indicates an NDM_RESET? this bit was wiped out, but no bit was added for hardware requests. This seems wrong...

Reply:
it was added for hw resets. Hw resets is now wider.

Reply:
Got it.

Reply:
something is very wrong. Let me double check and get back to you.

Reply:
ah! i remember why not. This is actually an artifact from the previous PR #15602 
You can see in that PR, the reset width actually "[grew](https://github.com/lowRISC/opentitan/pull/15602/files#diff-c167f43ab3134fa097e2e94eef43c13d14a483750c81da52d45e47f90ecd0dd5R1423)"  This isn't right because the previous PR effectively duplicated `ndm_reset`.  It had a named entry, AND it was part of the `hw_resets`.  So a result the previous `9 bits` was actually wrong, it should have been `8 bits`.  This PR basically corrects that again. 

Phew! I got very concerned for a moment there.

Reply:
Thanks for clarifying it.